### DISCUSSION TITLE
Chat handover from bot to live agent

### DISCUSSION THREAD

**Discussion by NitinNR** [chatwoot/chatwoot#4781] [Self hosted]
We are using Self-Hosted account.
We have implemented dialog flow for live chat agent. On selection of  specific option like "Chat with Sales" we want bot should stop and assign the chat to live agent. How we can configure both

**Comment by sojan-official** [comment]
ref: https://www.chatwoot.com/docs/product/channels/live-chat/integrations/dialogflow#creating-a-handoff-intent

**Reply by NitinNR** [reply]
whevever that intent gets run that chat (from website) should be assign to one of the online agents ?

**Reply by sojan-official** [reply]
if you have automatic assignment enabled, yes. the system will turn conversation status to open.
